src/report/graphs.py

```diff
@@ -843,99 +1064,131 @@ def plot(self, filename, size=1.0):
         # Override default figsize (make square), then scale by size parameter
         fig_width = fig_height = 4.0 * size
-        plt.rcParams.update({'figure.figsize': [fig_width, fig_height]})
+        plt.rcParams.update({"figure.figsize": [fig_width, fig_height]})
         # Minimize whitespace around chart
         extent = mpl.transforms.Bbox(((0, 0), (fig_width, fig_height)))
 
-        labels = '', ''
+        labels = "", ""
         sizes = [100 - self.percentage_full, self.percentage_full]
-        colors = ['white', self.fill_color]
+        colors = ["white", self.fill_color]
 
         # Set edge color to black
         #
         # See
         # https://matplotlib.org/users/dflt_style_changes.html#patch-edges-and-color
-        plt.rcParams['patch.force_edgecolor'] = True
-        plt.rcParams['patch.facecolor'] = 'b'
+        plt.rcParams["patch.force_edgecolor"] = True
+        plt.rcParams["patch.facecolor"] = "b"
 
         # autopct='%1.1f%%'
-        plt.pie(sizes, labels=labels, colors=colors, shadow=False,
-                startangle=90)
+        plt.pie(sizes, labels=labels, colors=colors, shadow=False, startangle=90)
 
         # Draw a circle at the center of pie to make it look like a donut
-        centre_circle = plt.Circle((0, 0), 0.75, color='black', fc='white',
-                                   linewidth=1.25)
+        centre_circle = plt.Circle(
+            (0, 0), 0.75, color="black", fc="white", linewidth=1.25
+        )
         fig = plt.gcf()
         fig.gca().add_artist(centre_circle)
 
-        plt.text(0, 0.15, str(self.percentage_full) + '%',
-                 horizontalalignment='center', verticalalignment='center',
-                 fontsize=50)
-        plt.text(0, -0.3, self.label,
-                 horizontalalignment='center',
-                 verticalalignment='center',
-                 fontsize=19.5, fontweight='bold')
+        plt.text(
+            0,
+            0.15,
+            str(self.percentage_full) + "%",
+            horizontalalignment="center",
+            verticalalignment="center",
+            fontsize=50,
+        )
+        plt.text(
+            0,
+            -0.3,
+            self.label,
+            horizontalalignment="center",
+            verticalalignment="center",
+            fontsize=19.5,
+            fontweight="bold",
+        )
 
         # Set aspect ratio to be equal so that pie is drawn as a circle.
-        plt.axis('equal')
+        plt.axis("equal")
         # plt.show()
         plt.tight_layout()  # trims margins down nicely
-        plt.savefig(filename + '.pdf', bbox_inches=extent, pad_inches=0)
+        plt.savefig(filename + ".pdf", bbox_inches=extent, pad_inches=0)
         plt.close()
 
 
 if __name__ == "__main__":
     setup()
 
-    m = MyMessage('Figure Omitted\nNo Vulnerabilities Detected')
-    m.plot('message')
+    m = MyMessage("Figure Omitted\nNo Vulnerabilities Detected")
+    m.plot("message")
```

Comment:
Since you removed the executable bit for this file is this bit still needed?

Reply:
See commit 28a45ff.